lib/improver/threshold.py

```diff
@@ -84,25 +86,31 @@ def __str__(self):
             'below_thresh_ok: {}>'
         ).format(self.threshold, self.fuzzy_factor, self.below_thresh_ok)
 
-    def process(self, cube):
+    def process(self, input_cube):
         """Convert each point to a truth value based on threshold. The truth
         value may or may not be fuzzy depending upon if a fuzzy_factor is
         supplied.
 
         Args:
-            cube : iris.cube.Cube
+            input_cube : iris.cube.Cube
                 Cube to threshold. The code is dimension-agnostic.
 
         Returns:
             cube : iris.cube.Cube
                 Cube after a threshold has been applied. The data within this
                 cube will contain values between 0 and 1 to indicate whether
                 a given threshold has been exceeded or not.
+                The cube attributes will contain:
+                  input_cube name prepended with "probability_of_"
```

Comment:
We could have some `* ` here to make it in into a bullet points, which will render better.